index.js

```diff
@@ -274,39 +274,52 @@ client.on(Events.InteractionCreate, async (interaction) => {
                 // Format review to be added
                 const review = `- ${reviewMessageDate}: ${messageLink}`;
 
+                const cleanMessageDate = CLEAN_MESSAGE_DATE ? CLEAN_MESSAGE_DATE : interaction.options.get('clean-date')?.value;
```

Comment:
A challenge for you - there's a way to write this without writing `CLEAN_MESSAGE_DATE`. Here's a hint: it's one of the boolean operators

On a sidenote, that trick is something I've only seen the _best_ JS developers (5+ years in JS) use, and that'll be you soon! 👀